scripts/buildExamples.js

```diff
@@ -1,46 +1,51 @@
 
-const fs = require('fs')
-const path = require('path')
+const { promises: fs } = require('fs');
+const path = require('path');
 
-const { bundle } = require('../src/build')
+const { bundle } = require('../src/build');
 
-const EXAMPLES_PATH = 'examples'
+const EXAMPLES_PATH = 'examples';
 
 fs.readdir(EXAMPLES_PATH, (err, results) => {
 
-  if (err) throw err
+  if (err) {
+    throw err;
+  }
 
-  results.forEach(examplesFile => {
+  results.forEach((examplesFile) => {
 
-    examplesFile = path.resolve(EXAMPLES_PATH, examplesFile)
+    let exampleFile = examplesFiles;
+    exampleFile = path.resolve(EXAMPLES_PATH, examplesFile);
 
-    fs.stat(examplesFile, (err, stat) => {
+    fs.stat(exampleFile, (error, stat) => {
 
-      if (err) throw err
+      if (error) {
+        throw error;
+      }
 
-      if (stat && stat.isDirectory()) {
+      if (stat?.isDirectory()) {
 
         try {
 
-          const srcPath = path.resolve(examplesFile, 'index.js')
-          const pkgPath = path.resolve(examplesFile, 'package.json')
-          const pkgStat = fs.statSync(pkgPath)
-          const srcStat = fs.statSync(srcPath)
+          const srcPath = path.resolve(exampleFile, 'index.js');
+          const pkgPath = path.resolve(exampleFile, 'package.json');
+          const pkgStat = await fs.stat(pkgPath);
+          const srcStat = await fs.stat(srcPath);
 
-          if (pkgStat && pkgStat.isFile() && srcStat && srcStat.isFile()) {
+          if (pkgStat?.isFile() && srcStat && srcStat.isFile()) {
```

Comment:
More optional chaining:
```suggestion
          if (pkgStat?.isFile() && srcStat?.isFile()) {
```